app/controllers/fitter_happier/fitter_happier_controller.rb

```diff
@@ -0,0 +1,30 @@
+module FitterHappier
+  class FitterHappierController < ActionController::Base
```

Comment:
Not a fan of the double `FitterHappier` naming. Perhaps `FitterHappier::HeartbeatController` or something would be a better name.